app/_hub/justeat/kongverge/index.md

```diff
@@ -0,0 +1,77 @@
+---
+name: Kongverge
+publisher: Just Eat
+
+categories:
+  - deployment
+
+type: integration
+
+desc: A Desired State Configuration tool for Kong.
+
+description: | 
+  Kongverge is a command-line tool written in .NET Core 2.1. It is used to configure a kong server by moving its state into sync with the “desired state” given in configuration files.
+
+  Kongverge uses several ‘data transfer objects’ to read from files and write to Kong (and vice versa). For simplicity, the field names on these objects generally match what is present in Kong:
+
+  * KongConfiguration
+  * KongRoute 
+  * KongService 
+  * KongPlugin
+
+  These objects also handle matching - i.e. reconciling the state described by files with the state in Kong, and performing actions in Kong as needed to make them the same. The possible cases for these objects are:
+
+  * Unchanged; The object in Kong is identical to the object in config, so no action is required.
+  * Changed; the object in Kong is matched with an object in config, but not all of the properties are  the same. Action is required to update the object in place.
+  * New; the object needs to be added to Kong.
+  * Deleted; the object needs to be removed from Kong.
+
+support_url: https://github.com/justeat/kongverge/issues
+
+source_url: https://github.com/justeat/kongverge
+
+license_type: Apache-2.0
+
+license_url: https://github.com/justeat/kongverge/blob/master/LICENSE
+
+kong_version_compatibility:
+    community_edition:
+      compatible:
+        - 0.14.x
+#      incompatible:
+#        - 0.13.x
+#        - 0.12.x
+#        - 0.11.x
+#        - 0.10.x
+#        - 0.9.x
+#        - 0.8.x
+#        - 0.7.x
+#        - 0.6.x
+#        - 0.5.x
+#        - 0.4.x
+#        - 0.3.x
+#        - 0.2.x
+    enterprise_edition:
+      compatible:
+        - 0.33-x
```

Comment:
I just want to check on this statement of compatibility with Kong Enterprise 0.33 - that version is based on Kong 0.13.1 per https://docs.konghq.com/enterprise/changelog/#033---20180724 - and I see above that you don't declare compatibility with Kong 0.13.x.

Can you clarify with what versions of Kong and Kong Enterprise you've successfully used Kongverge? Thanks!

Reply:
Thanks for your reply! 

Kongverge is compatible with CE 0.14.0 onwards, we have tested with 0.13.0 & 1 and some of our unit tests fail. I have checked the change log documentation for Kong Enterprise versions and the latest version (0.33-2) inherits from CE 0.13.0 & 1, does this render Kongverge incompatible with any Kong EE versions?

Hope this makes sense!

Reply:
Yes, I believe that Kongverge would currently be incompatible with all released versions of Kong Enterprise. Future Kong Enterprise versions will, eventually, be based on versions of Kong with which Kongverge is compatible.

So, I'd suggest you mark all versions of Kong prior to 0.14, and all versions of Kong Enterprise, as incompatible - thanks.

Reply:
Okay, thank you for your help. I have updated and committed the changes.